[2022-02-28 20:04:41,471] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:04:45,036] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 20:04:45,038] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2167
[2022-02-28 20:04:45,283] [log.py:230] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-02-28 20:04:45,284] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2218
[2022-02-28 20:07:08,788] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:07:09,873] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:08:00,019] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 20:08:00,021] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2273
[2022-02-28 20:08:13,953] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/
[2022-02-28 20:08:13,954] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/ HTTP/1.1" 404 2379
[2022-02-28 20:08:38,449] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20215
[2022-02-28 20:08:38,479] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/gloab.css HTTP/1.1" 200 16102
[2022-02-28 20:08:38,480] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 200 6224
[2022-02-28 20:08:38,482] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/yzm1.jpg HTTP/1.1" 200 2399
[2022-02-28 20:08:38,483] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 200 10854
[2022-02-28 20:08:38,485] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/bootstrap.min.css HTTP/1.1" 200 150049
[2022-02-28 20:08:38,486] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 20:10:53,854] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:10:55,999] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 20:10:56,003] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2273
[2022-02-28 20:10:58,685] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 20:10:58,687] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2273
[2022-02-28 20:11:47,451] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/index/
[2022-02-28 20:11:47,452] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/index/ HTTP/1.1" 404 2397
[2022-02-28 20:12:06,672] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contenst
[2022-02-28 20:12:06,673] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contenst HTTP/1.1" 404 2403
[2022-02-28 20:12:19,578] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/contents/ HTTP/1.1" 200 128061
[2022-02-28 20:12:19,629] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 200 1609
[2022-02-28 20:12:19,631] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/master.css HTTP/1.1" 200 7479
[2022-02-28 20:12:19,632] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/common.css HTTP/1.1" 200 7892
[2022-02-28 20:12:19,633] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/strip.css HTTP/1.1" 200 12521
[2022-02-28 20:12:19,636] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/jquery.easy_slides.css HTTP/1.1" 200 10445
[2022-02-28 20:12:19,640] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/normalize.css HTTP/1.1" 200 2503
[2022-02-28 20:12:19,641] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/htmleaf-demo.css HTTP/1.1" 200 5731
[2022-02-28 20:12:19,645] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/default.css HTTP/1.1" 200 5993
[2022-02-28 20:12:19,646] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/logo.png HTTP/1.1" 200 4832
[2022-02-28 20:12:19,652] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/01.jpg HTTP/1.1" 200 17291
[2022-02-28 20:12:19,658] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/02.jpg HTTP/1.1" 200 22664
[2022-02-28 20:12:19,670] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/03.jpg HTTP/1.1" 200 28024
[2022-02-28 20:12:19,671] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/04.jpg HTTP/1.1" 200 29823
[2022-02-28 20:12:19,675] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/06.jpg HTTP/1.1" 200 37717
[2022-02-28 20:12:19,676] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/05.jpg HTTP/1.1" 200 34378
[2022-02-28 20:12:19,680] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/07.jpg HTTP/1.1" 200 28285
[2022-02-28 20:12:19,696] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jssor.slider-21.1.6.min.js HTTP/1.1" 200 113520
[2022-02-28 20:12:19,696] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/index.js HTTP/1.1" 200 10938
[2022-02-28 20:12:19,696] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.easy_slides.js HTTP/1.1" 200 14760
[2022-02-28 20:12:19,697] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/elevator.min.js HTTP/1.1" 200 1940
[2022-02-28 20:12:19,697] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/common.js HTTP/1.1" 200 10896
[2022-02-28 20:12:19,698] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/pinterest_grid.js HTTP/1.1" 200 4261
[2022-02-28 20:12:19,713] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/animatescroll.min.js HTTP/1.1" 200 4144
[2022-02-28 20:12:19,714] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/09.jpg HTTP/1.1" 200 11378
[2022-02-28 20:12:19,716] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/10.jpg HTTP/1.1" 200 23869
[2022-02-28 20:12:19,718] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/08.jpg HTTP/1.1" 200 34638
[2022-02-28 20:12:19,718] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/11.jpg HTTP/1.1" 200 24199
[2022-02-28 20:12:19,722] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/strip.pkgd.min.js HTTP/1.1" 200 68596
[2022-02-28 20:12:19,726] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/12.jpg HTTP/1.1" 200 27326
[2022-02-28 20:12:19,737] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/%E7%83%AD%E7%88%B1%E7%94%9F%E6%B4%BB%E7%BE%8E%E5%A5%BD%E7%94%9F%E6%B4%BB.png HTTP/1.1" 200 6365
[2022-02-28 20:12:19,740] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/01.jpg HTTP/1.1" 200 17291
[2022-02-28 20:12:19,740] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/14.jpg HTTP/1.1" 200 30980
[2022-02-28 20:12:19,743] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/13.jpg HTTP/1.1" 200 40307
[2022-02-28 20:12:19,747] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/pub_img/2.jpg HTTP/1.1" 200 36908
[2022-02-28 20:12:19,749] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/pub_img/1.jpg HTTP/1.1" 200 42385
[2022-02-28 20:12:19,764] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/02.jpg HTTP/1.1" 200 22664
[2022-02-28 20:12:19,765] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/pub_img/3.jpg HTTP/1.1" 200 70188
[2022-02-28 20:12:19,765] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/03.jpg HTTP/1.1" 200 28024
[2022-02-28 20:12:19,765] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/05.jpg HTTP/1.1" 200 34378
[2022-02-28 20:12:19,770] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/04.jpg HTTP/1.1" 200 29823
[2022-02-28 20:12:19,787] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/260_200.png HTTP/1.1" 200 5494
[2022-02-28 20:12:19,787] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/06.jpg HTTP/1.1" 200 37717
[2022-02-28 20:12:19,789] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/09.jpg HTTP/1.1" 200 11378
[2022-02-28 20:12:19,791] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/img/08.jpg HTTP/1.1" 200 34638
[2022-02-28 20:12:19,791] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/upload/img/07.jpg%20%%7D HTTP/1.1" 404 1825
[2022-02-28 20:12:19,817] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/%E6%98%A5%E8%8A%82.png HTTP/1.1" 200 264286
[2022-02-28 20:12:19,819] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/contents/ HTTP/1.1" 200 128061
[2022-02-28 20:12:19,822] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/upload/600_400.png HTTP/1.1" 200 6221
[2022-02-28 20:12:19,824] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/t01.png HTTP/1.1" 200 1621
[2022-02-28 20:12:19,826] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/static/img/img_nav/loading.gif
[2022-02-28 20:12:19,828] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/a17.png HTTP/1.1" 200 5512
[2022-02-28 20:12:19,830] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/static/img/img_nav/loading.gif HTTP/1.1" 404 2469
[2022-02-28 20:12:19,964] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.woff?rretjt HTTP/1.1" 200 13412
[2022-02-28 20:12:20,106] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,110] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:12:20,111] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/close.svg HTTP/1.1" 200 4963
[2022-02-28 20:12:20,125] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,127] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:12:20,180] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E5%BE%AE%E8%BD%AF%E7%AE%80%E6%A5%B7%E4%BD%93.ttf HTTP/1.1" 200 4511764
[2022-02-28 20:12:20,287] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E6%80%9D%E6%BA%90%E5%AE%8B%E4%BD%93.ttf HTTP/1.1" 200 11772444
[2022-02-28 20:12:20,623] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,625] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 20:12:20,627] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:12:20,646] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,649] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,650] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:12:20,652] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:12:20,663] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/true
[2022-02-28 20:12:20,665] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/true HTTP/1.1" 404 2418
[2022-02-28 20:13:07,778] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:13:09,099] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:13:21,587] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\contents\urls.py changed, reloading.
[2022-02-28 20:13:22,684] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:13:23,136] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/contents/
[2022-02-28 20:13:23,137] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/contents/ HTTP/1.1" 404 2391
[2022-02-28 20:13:30,010] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/
[2022-02-28 20:13:30,011] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/ HTTP/1.1" 404 2364
[2022-02-28 20:13:35,662] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 128061
[2022-02-28 20:13:35,723] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/upload/img/07.jpg%20%%7D HTTP/1.1" 404 1825
[2022-02-28 20:13:35,742] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 128061
[2022-02-28 20:13:35,744] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/img_nav/loading.gif HTTP/1.1" 200 4765
[2022-02-28 20:13:35,854] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,856] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:13:35,863] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,865] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:13:35,941] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,943] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,945] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:13:35,945] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:13:35,960] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,963] [log.py:230] [log:log_response] [WARNING]- Not Found: /true
[2022-02-28 20:13:35,965] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:13:35,967] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /true HTTP/1.1" 404 2358
[2022-02-28 20:15:28,934] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 20:15:30,136] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:15:36,890] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20215
[2022-02-28 20:15:37,787] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/showPwd.png HTTP/1.1" 200 416
[2022-02-28 20:15:39,610] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/fonts/icomoon.woff?rretjt HTTP/1.1" 200 1452
[2022-02-28 20:15:39,627] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.ttf?7kkyc2 HTTP/1.1" 200 13336
[2022-02-28 20:16:51,285] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
KeyError: 'contente'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 38, in post
    return redirect(reverse('contente:index'))
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'contente' is not a registered namespace
[2022-02-28 20:16:51,293] [basehttp.py:161] [basehttp:log_message] [ERROR]- "POST /users/register/ HTTP/1.1" 500 93688
[2022-02-28 20:43:06,901] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:43:08,098] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:44:21,873] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:44:22,800] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:45:40,185] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:45:41,113] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:46:22,883] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:46:23,870] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:57:06,297] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:57:07,335] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 20:57:29,230] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 20:57:30,633] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:01:25,587] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 21:01:26,645] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:03:05,029] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 21:03:06,262] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:12:11,166] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:12:59,712] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 21:13:00,863] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:15:30,775] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 21:15:31,930] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:16:30,227] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 21:16:31,409] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:17:09,923] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 21:17:11,215] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:17:36,930] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 21:17:38,031] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:25:33,943] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register HTTP/1.1" 301 0
[2022-02-28 21:25:34,260] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 12, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 21:25:34,276] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 138519
[2022-02-28 21:25:36,309] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 12, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 21:25:36,317] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 138656
[2022-02-28 21:31:53,507] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 21:31:54,540] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:32:16,089] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 13, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 21:32:16,098] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 138360
[2022-02-28 21:42:53,159] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 21:42:54,148] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:43:59,846] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\urls.py changed, reloading.
[2022-02-28 21:44:00,965] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:44:12,020] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\contents\urls.py changed, reloading.
[2022-02-28 21:44:13,051] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:45:33,628] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 21:45:34,632] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:48:17,825] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 21:48:17,826] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2335
[2022-02-28 21:48:30,365] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 21:48:30,366] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2335
[2022-02-28 21:48:35,252] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 13, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 21:48:35,261] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 138649
[2022-02-28 21:49:29,009] [log.py:230] [log:log_response] [WARNING]- Not Found: /users/login/
[2022-02-28 21:49:29,010] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /users/login/ HTTP/1.1" 404 2636
[2022-02-28 21:50:27,526] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 21:50:28,494] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:50:32,986] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 21:50:33,015] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/login.css HTTP/1.1" 200 3263
[2022-02-28 21:50:33,016] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/logo.png HTTP/1.1" 200 4832
[2022-02-28 21:50:33,050] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/gVerify.js HTTP/1.1" 200 4451
[2022-02-28 21:50:33,052] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/bootstrap.min.js HTTP/1.1" 200 39680
[2022-02-28 21:50:33,054] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/hy.png HTTP/1.1" 200 1582
[2022-02-28 21:50:33,060] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/layer.js HTTP/1.1" 200 36155
[2022-02-28 21:50:33,064] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/suo.png HTTP/1.1" 200 1278
[2022-02-28 21:50:33,066] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg2.jpg HTTP/1.1" 200 26904
[2022-02-28 21:50:33,068] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 21:50:33,071] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg1.jpg HTTP/1.1" 200 40291
[2022-02-28 21:50:33,098] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/theme/default/layer.css?v=3.5.1 HTTP/1.1" 200 14271
[2022-02-28 21:51:46,936] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\contents\urls.py changed, reloading.
[2022-02-28 21:51:47,917] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 21:51:56,193] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 128061
[2022-02-28 21:51:56,309] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 128061
[2022-02-28 21:51:56,314] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/upload/img/07.jpg%20%%7D HTTP/1.1" 404 1825
[2022-02-28 21:51:56,536] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,537] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:51:56,544] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,545] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:51:56,943] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,944] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:51:56,958] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,961] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,962] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:51:56,963] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:51:56,973] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 21:51:56,975] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 21:52:10,140] [log.py:230] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-02-28 21:52:10,141] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2385
[2022-02-28 21:53:22,790] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 13, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 21:53:22,802] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 135658
[2022-02-28 21:53:47,812] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20195
[2022-02-28 22:50:23,943] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 22:50:24,911] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:16:51,979] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 23:16:53,026] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:16:59,249] [log.py:230] [log:log_response] [ERROR]- Internal Server Error: /users/register/
Traceback (most recent call last):
  File "D:\Python37\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Python37\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Django_aijia\my_aijia\my_aijia/apps\users\views.py", line 13, in get
    return render(request, 'register.html')
  File "D:\Python37\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Python37\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "D:\Python37\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "D:\Python37\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "D:\Python37\lib\site-packages\django\template\defaulttags.py", line 446, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Python37\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Python37\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login' not found. 'login' is not a valid view function or pattern name.
[2022-02-28 23:16:59,258] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /users/register/ HTTP/1.1" 500 135352
[2022-02-28 23:16:59,430] [log.py:230] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-02-28 23:16:59,431] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2385
[2022-02-28 23:20:29,934] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20201
[2022-02-28 23:20:29,974] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 304 0
[2022-02-28 23:20:29,998] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 304 0
[2022-02-28 23:21:54,650] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:21:55,619] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:22:12,364] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20207
[2022-02-28 23:22:12,407] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 200 6224
[2022-02-28 23:22:12,408] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/gloab.css HTTP/1.1" 200 16102
[2022-02-28 23:22:12,413] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/bootstrap.min.css HTTP/1.1" 200 150049
[2022-02-28 23:22:12,446] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 200 10854
[2022-02-28 23:22:12,446] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/yzm1.jpg HTTP/1.1" 200 2399
[2022-02-28 23:22:12,451] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:22:12,457] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/showPwd.png HTTP/1.1" 200 416
[2022-02-28 23:22:12,617] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E6%80%9D%E6%BA%90%E5%AE%8B%E4%BD%93.ttf HTTP/1.1" 200 11772444
[2022-02-28 23:22:12,825] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:22:14,213] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/fonts/icomoon.woff?rretjt HTTP/1.1" 200 1452
[2022-02-28 23:22:14,251] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.ttf?7kkyc2 HTTP/1.1" 200 13336
[2022-02-28 23:22:47,945] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 302 0
[2022-02-28 23:22:47,956] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/ HTTP/1.1" 200 3942
[2022-02-28 23:24:15,637] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:24:16,590] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:27:03,562] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20207
[2022-02-28 23:28:33,644] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:28:34,747] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:29:11,698] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
[2022-02-28 23:29:12,711] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:29:16,489] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20219
[2022-02-28 23:29:16,526] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/gloab.css HTTP/1.1" 200 16102
[2022-02-28 23:29:16,527] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 200 6224
[2022-02-28 23:29:16,530] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/bootstrap.min.css HTTP/1.1" 200 150049
[2022-02-28 23:29:16,546] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/yzm1.jpg HTTP/1.1" 200 2399
[2022-02-28 23:29:16,546] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 200 10854
[2022-02-28 23:29:16,549] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:29:16,556] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/showPwd.png HTTP/1.1" 200 416
[2022-02-28 23:29:16,649] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E6%80%9D%E6%BA%90%E5%AE%8B%E4%BD%93.ttf HTTP/1.1" 200 11772444
[2022-02-28 23:29:16,843] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:29:18,278] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/fonts/icomoon.woff?rretjt HTTP/1.1" 200 1452
[2022-02-28 23:29:18,320] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.ttf?7kkyc2 HTTP/1.1" 200 13336
[2022-02-28 23:30:06,272] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 302 0
[2022-02-28 23:30:06,305] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 23:31:02,614] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:31:03,652] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:46:45,320] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 23:46:45,356] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/login.css HTTP/1.1" 200 3263
[2022-02-28 23:46:45,357] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/logo.png HTTP/1.1" 200 4832
[2022-02-28 23:46:45,408] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/bootstrap.min.js HTTP/1.1" 200 39680
[2022-02-28 23:46:45,410] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/gVerify.js HTTP/1.1" 200 4451
[2022-02-28 23:46:45,420] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:46:45,422] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg2.jpg HTTP/1.1" 200 26904
[2022-02-28 23:46:45,422] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/hy.png HTTP/1.1" 200 1582
[2022-02-28 23:46:45,425] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg1.jpg HTTP/1.1" 200 40291
[2022-02-28 23:46:45,427] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/suo.png HTTP/1.1" 200 1278
[2022-02-28 23:46:45,427] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/layer.js HTTP/1.1" 200 36155
[2022-02-28 23:46:45,453] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/theme/default/layer.css?v=3.5.1 HTTP/1.1" 200 14271
[2022-02-28 23:46:45,553] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:46:47,287] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 23:46:47,319] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/login.css HTTP/1.1" 200 3263
[2022-02-28 23:46:47,319] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/logo.png HTTP/1.1" 200 4832
[2022-02-28 23:46:47,343] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/gVerify.js HTTP/1.1" 200 4451
[2022-02-28 23:46:47,345] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/suo.png HTTP/1.1" 200 1278
[2022-02-28 23:46:47,346] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/bootstrap.min.js HTTP/1.1" 200 39680
[2022-02-28 23:46:47,347] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/hy.png HTTP/1.1" 200 1582
[2022-02-28 23:46:47,354] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/layer.js HTTP/1.1" 200 36155
[2022-02-28 23:46:47,357] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg2.jpg HTTP/1.1" 200 26904
[2022-02-28 23:46:47,358] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/login/bg1.jpg HTTP/1.1" 200 40291
[2022-02-28 23:46:47,359] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:46:47,377] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/login/layer-v3.5.1/theme/default/layer.css?v=3.5.1 HTTP/1.1" 200 14271
[2022-02-28 23:46:47,404] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:46:54,488] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20399
[2022-02-28 23:47:37,818] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20364
[2022-02-28 23:49:11,339] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:49:12,294] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:49:27,848] [log.py:230] [log:log_response] [WARNING]- Not Found: /
[2022-02-28 23:49:27,849] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2334
[2022-02-28 23:49:28,035] [log.py:230] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-02-28 23:49:28,036] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2385
[2022-02-28 23:49:40,766] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index HTTP/1.1" 301 0
[2022-02-28 23:49:40,787] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 128061
[2022-02-28 23:49:40,827] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/master.css HTTP/1.1" 304 0
[2022-02-28 23:49:40,906] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /index/ HTTP/1.1" 200 128061
[2022-02-28 23:49:40,909] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/upload/img/07.jpg%20%%7D HTTP/1.1" 404 1825
[2022-02-28 23:49:41,138] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,139] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:41,148] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,149] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:41,477] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,478] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:41,494] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,496] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,497] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:41,498] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:41,507] [log.py:230] [log:log_response] [WARNING]- Not Found: /index/true
[2022-02-28 23:49:41,509] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /index/true HTTP/1.1" 404 2382
[2022-02-28 23:49:45,959] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20399
[2022-02-28 23:50:37,902] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 302 0
[2022-02-28 23:50:37,927] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 23:51:00,523] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:51:01,540] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:51:02,947] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20399
[2022-02-28 23:51:32,111] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 302 0
[2022-02-28 23:51:32,138] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3942
[2022-02-28 23:51:37,574] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20399
[2022-02-28 23:51:58,769] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20364
[2022-02-28 23:52:22,036] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20364
[2022-02-28 23:52:56,847] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\views.py changed, reloading.
[2022-02-28 23:52:57,843] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-02-28 23:53:24,994] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20062
[2022-02-28 23:53:25,068] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/gloab.css HTTP/1.1" 200 16102
[2022-02-28 23:53:25,070] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 200 6224
[2022-02-28 23:53:25,071] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/bootstrap.min.css HTTP/1.1" 200 150049
[2022-02-28 23:53:25,122] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/yzm1.jpg HTTP/1.1" 200 2399
[2022-02-28 23:53:25,124] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 200 10854
[2022-02-28 23:53:25,126] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/showPwd.png HTTP/1.1" 200 416
[2022-02-28 23:53:25,131] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:53:25,282] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E6%80%9D%E6%BA%90%E5%AE%8B%E4%BD%93.ttf HTTP/1.1" 200 11772444
[2022-02-28 23:53:25,476] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:53:25,874] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/fonts/icomoon.woff?rretjt HTTP/1.1" 200 1452
[2022-02-28 23:53:25,887] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.ttf?7kkyc2 HTTP/1.1" 200 13336
[2022-02-28 23:54:05,019] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20062
[2022-02-28 23:56:40,887] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /users/register/ HTTP/1.1" 200 20074
[2022-02-28 23:56:40,938] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/register.css HTTP/1.1" 200 6224
[2022-02-28 23:56:40,938] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/gloab.css HTTP/1.1" 200 16102
[2022-02-28 23:56:40,941] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/css/bootstrap.min.css HTTP/1.1" 200 150049
[2022-02-28 23:56:40,957] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/yzm1.jpg HTTP/1.1" 200 2399
[2022-02-28 23:56:40,958] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/register.js HTTP/1.1" 200 10854
[2022-02-28 23:56:40,960] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 200 154528
[2022-02-28 23:56:40,967] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/img/register/showPwd.png HTTP/1.1" 200 416
[2022-02-28 23:56:41,056] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/%E6%80%9D%E6%BA%90%E5%AE%8B%E4%BD%93.ttf HTTP/1.1" 200 11772444
[2022-02-28 23:56:41,250] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/favicon.ico HTTP/1.1" 200 4286
[2022-02-28 23:56:41,851] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/fonts/icomoon.woff?rretjt HTTP/1.1" 200 1452
[2022-02-28 23:56:41,871] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/fonts/icomoon.ttf?7kkyc2 HTTP/1.1" 200 13336
[2022-02-28 23:56:41,976] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3957
[2022-02-28 23:56:46,844] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3957
[2022-02-28 23:56:50,069] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3957
[2022-02-28 23:56:52,982] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/login/ HTTP/1.1" 200 3957
[2022-02-28 23:56:56,244] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /users/register/ HTTP/1.1" 200 20427
[2022-03-01 00:03:35,961] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- D:\Django_aijia\my_aijia\my_aijia\apps\users\urls.py changed, reloading.
